src/components/swap/AdvancedSwapDetails.tsx

```diff
@@ -45,119 +39,92 @@ function TextWithLoadingPlaceholder({
-            </ThemedText.DeprecatedBlack>
+            <ThemedText.BodySmall>~${trade.gasUseEstimateUSD.toFixed(2)}</ThemedText.BodySmall>
           </TextWithLoadingPlaceholder>
         </RowBetween>
-        {!trade?.gasUseEstimateUSD || !chainId || !SUPPORTED_GAS_ESTIMATE_CHAIN_IDS.includes(chainId) ? null : (
-          <RowBetween>
-            <MouseoverTooltip
-              text={
-                <Trans>
-                  The fee paid to miners who process your transaction. This must be paid in {nativeCurrency.symbol}.
-                </Trans>
-              }
-              disableHover={hideInfoTooltips}
-            >
-              <ThemedText.DeprecatedSubHeader color={theme.textTertiary}>
-                <Trans>Network Fee</Trans>
-              </ThemedText.DeprecatedSubHeader>
-            </MouseoverTooltip>
-            <TextWithLoadingPlaceholder syncing={syncing} width={50}>
-              <ThemedText.DeprecatedBlack textAlign="right" fontSize={14} color={theme.textTertiary}>
-                ~${trade.gasUseEstimateUSD.toFixed(2)}
-              </ThemedText.DeprecatedBlack>
-            </TextWithLoadingPlaceholder>
-          </RowBetween>
-        )}
-      </AutoColumn>
-    </StyledCard>
+      )}
+      <RowBetween>
+        <RowFixed>
+          <MouseoverTooltip
+            text={
+              <Trans>
+                The minimum amount you are guaranteed to receive. If the price slips any further, your transaction will
+                revert.
+              </Trans>
+            }
+          >
+            <ThemedText.BodySmall color="textSecondary">
+              <Trans>Minimum output</Trans>
+            </ThemedText.BodySmall>
+          </MouseoverTooltip>
+        </RowFixed>
+        <TextWithLoadingPlaceholder syncing={syncing} width={70}>
+          <ThemedText.BodySmall>
+            {trade.tradeType === TradeType.EXACT_INPUT
+              ? `${trade.minimumAmountOut(allowedSlippage).toSignificant(6)} ${trade.outputAmount.currency.symbol}`
+              : `${trade.maximumAmountIn(allowedSlippage).toSignificant(6)} ${trade.inputAmount.currency.symbol}`}
+          </ThemedText.BodySmall>
+        </TextWithLoadingPlaceholder>
+      </RowBetween>
+      <RowBetween>
+        <RowFixed>
+          <MouseoverTooltip
+            text={
+              <Trans>
+                The amount you expect to receive at the current market price. You may receive less or more if the market
+                price changes while your transaction is pending.
+              </Trans>
+            }
+          >
+            <ThemedText.BodySmall color="textSecondary">
+              <Trans>Expected output</Trans>
+            </ThemedText.BodySmall>
+          </MouseoverTooltip>
+        </RowFixed>
+        <TextWithLoadingPlaceholder syncing={syncing} width={65}>
+          <ThemedText.BodySmall>
+            {`${trade.outputAmount.toSignificant(6)} ${trade.outputAmount.currency.symbol}`}
```

Comment:
nit: how should we treat formatting? i know we have a function some places but not all

Reply:
we should do a full refresh of swap flow formatting in a later proj!

Reply:
Agreed, based on experience working on mobile, this way is not the beast for many reasons